package.json

```diff
@@ -69,12 +61,14 @@
     "@typescript-eslint/eslint-plugin": "^2.18.0",
     "@typescript-eslint/parser": "^2.18.0",
     "concurrently": "^5.0.0",
+    "expo-yarn-workspaces": "^1.2.1",
     "husky": "^4.2.1",
     "jest": "^25.1.0",
     "jest-cli": "25.1.0",
     "jest-junit": "^10.0.0",
     "lerna": "^3.20.2",
     "lint-staged": "^10.0.3",
+    "metro-config": "^0.58.0",
```

Comment:
do we need `metro-config` on the root?

Reply:
Yes, because if the metro config is only inside of expo app, the Expo service does not start